fix typo: "to many" to "too many"

halmai opened the pull request:
Obvious Fix

When the server responses `StatusTooManyRequests` (`429`), the SDK returned an error with a mistyped string:

```
return errors.New("to many requests")
```

This PR fixes the typo.

bretterer commented:
Thanks for the fix.  It appears that you reverted the change so there is nothing to commit here

halmai commented:
> Thanks for the fix. It appears that you reverted the change, so there is nothing to commit here.

I reverted it to see if my change caused the failure of PR. I saw later that the master was already broken, so now I undo the revert, rebased it and squashed everything into one simple commit. 

The travis check should pass when the master gets fixed.

bretterer commented:
No. its because travis uses private environment variables. If you don't have access to merge to the repo, travis will not provide access to those variables, meaning the tests will fail.

We will take a look and make sure it passes through a forked PR based on this one to get it merged in

halmai commented:
> We will take a look and make sure it passes through a forked PR based on this one to get it merged in

Thanks, @bretterer. 

Do you have any idea when it gets merged? We need to check this particular error in our code and I wouldn't like to compare the result to a mistyped string, if it is avoidable.

halmai commented:
> No. its because travis uses private environment variables. If you don't have access to merge to the repo, travis will not provide access to those variables, meaning the tests will fail.

I think this is a relevant information, probably it is worth mentioning in the `CONTRIBUTING.md` docs.

halmai commented:
Hi @bretterer, is there any progress with this PR? Could anyone run the tests with the appropriate permissions so that it can be merged?
